domain-management/src/main/java/org/jboss/as/domain/management/security/SecretIdentityService.java

```diff
@@ -66,7 +67,7 @@ public void start(StartContext startContext) throws StartException {
         final char[] thePassword;
         if (base64) {
             byte[] value = Base64.getDecoder().decode(password);
-            String tempPassword = new String(value);
+            String tempPassword = new String(value, StandardCharsets.UTF_8);
```

Comment:
Probably not a big deal, but the `Base64.getDecoder().decode(string)` uses ISO_8859_1 encoding by default. UTF-8 should work fine, I just wasn't sure if we wanted to be consistent with what the decoder uses.